crates/core_arch/src/x86/sse41.rs

```diff
@@ -155,11 +155,9 @@ pub unsafe fn _mm_blend_ps<const IMM4: i32>(a: __m128, b: __m128) -> __m128 {
 /// #       #[target_feature(enable = "sse4.1")]
 /// #       unsafe fn worker() {
 /// let mut float_store = vec![1.0, 1.0, 2.0, 3.0];
-/// unsafe {
 ///     let simd_floats = _mm_set_ps(2.5, 5.0, 7.5, 10.0);
 ///     let x: i32 = _mm_extract_ps::<2>(simd_floats);
 ///     float_store.push(f32::from_bits(x as u32));
-/// }
 /// assert_eq!(float_store, vec![1.0, 1.0, 2.0, 3.0, 5.0]);
 /// #       }
 /// #       unsafe { worker() }
```

Comment:
This appears "unused" from inside the source, but not to the user reading the docs, due to the unsafe "worker" function (that is required to use `target_feature(enable)`) it is nested inside, so if a user copies this code from the rendered docs with this change, they will need to readd the `unsafe { }`. That is to say: this makes the documentation less correct. This is an instance of `unused_unsafe` only because `fn worker` is not annotated `#[warn(unsafe_op_in_unsafe_fn)]`, which should also eliminate the warning, and without altering the correctness of the documentation.